docs/development/releasing.rst

```diff
@@ -412,22 +421,12 @@ clean-up tasks to finalize the process.
 Post-Release procedures
 -----------------------
 
-#. If this is a release of the current release (i.e., not an LTS supported along
-   side a more recent version), update the "stable" branch to point to the new
-   release::
+#. Update the "stable" branch to point to the new release::
 
       $ git checkout stable
       $ git reset --hard v<version>
       $ git push upstream stable --force
 
-#. If this is an LTS release (whether or not it is being supported alongside
```

Comment:
Does this mean we delete the LTS branch from repo after APE 21 is accepted?

Reply:
No because we might break old links I think? We can just leave the branch frozen?

Reply:
In my projects without a `stable` branch, the "stable" on RTD automagically picks up the latest release. 🤷 

Do you plan to delete the `stable` branch altogether?

Reply:
Right, then we should delete the stable branch.
https://docs.readthedocs.io/en/stable/versions.html

Reply:
When should I do that? Now or when you cut v6.0 ?

Reply:
Done. `stable` is gone.